README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+## **About me**
```

Comment:
Your headers (both the 'About me' and 'Fun facts') work well, and they look good! When I was doing mine, I found that the headers are automatically bolded, so I'm not sure that it is necessary to wrap the text in **. This might be something to try out!